modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -125,39 +129,67 @@ DEFAULT pxeboot
+Ignition config file on your HTTP server.
 
 ifndef::only-pxe[]
 ** For iPXE:
 +
 ----
-kernel  http://<HTTP_server>/rhcos-<version>-installer-kernel-<architecture> ip=dhcp rd.neednet=1 initrd=http://<HTTP_server>/rhcos-<version>-installer-initramfs.<architecture>.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://<HTTP_server>/rhcos-<version>-metal.<arhcitectutre>.raw.gz coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <1> <2>
-initrd http://<HTTP_server>/rhcos-<version>-installer-initramfs.<architecture>.img <3>
+kernel  http://<HTTP_server>/rhcos-<version>-<architecture>-installer-kernel-<architecture> ip=dhcp rd.neednet=1 initrd=rhcos-<version>-<architecture>-installer-initramfs.<architecture>.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://<HTTP_server>/rhcos-<version>-<architecture>-metal.<architecture>.raw.gz coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <1> <2>
+initrd http://<HTTP_server>/rhcos-<version>-<architecture>-installer-initramfs.<architecture>.img <3>
 boot
 ----
 <1> Specify locations of the {op-system} files that you uploaded to your
 HTTP server. The `kernel` parameter value is the location of the `kernel` file,
-the `initrd` parameter value is the location of the `initramfs` file,
-the `coreos.inst.image_url` parameter value is the location of the compressed
-metal RAW image, and the `coreos.inst.ignition_url` parameter value is the
-location of the bootstrap Ignition config file.
+the `initrd` parameter value references the name of the `initramfs` file that is
+supplied on the `initrd` line below, the `coreos.inst.image_url` parameter value
+is the location of the compressed metal RAW image, and the `coreos.inst.ignition_url`
+parameter value is the location of the bootstrap Ignition config file.
 <2> If you use multiple NICs, specify a single interface in the `ip` option.
 For example, to use DHCP on a NIC that is named `eno1`, set `ip=eno1:dhcp`.
 <3> Specify the location of the `initramfs` file that you uploaded to your HTTP
 server.
 endif::only-pxe[]
 
+. If you use UEFI, perform the following actions:
+.. Provide the EFI binaries and `grub.cfg` file that are required for booting the system. You need the `shim.efi` binary and the `grubx64.efi` binary. Use one of the following methods to obtain these files:
+
+** Retrieve these files from the `shim` and `grub2-efi` packages included with Red Hat Enterprise Linux 8.  See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_installation/preparing-for-a-network-install_installing-rhel-as-an-experienced-user#configuring-a-tftp-server-for-uefi-based-clients_preparing-for-a-network-install[RHEL8 instructions] for extracting the EFI binaries from RPMs. Then copy the `shim.efi` and `grubx64.efi` binaries to your TFTP server.
```

Comment:
Since we're asking the user to copy the `grub.cfg` from the ISO anyway, maybe we should drop this case?

Reply:
@kalexand-rh Do you think there is value in providing additional ways to accomplish the end goal or keep it simple?

Reply:
@miabbott, unless there's a clear use case for why you'd need a more complicated method, I think that it's best to present the simplest method.

Reply:
OK, let me remove it and push a final (:crossed_fingers:) update